lib/release_notes_bot_web/controllers/slack_interaction_controller.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ReleaseNotesBotWeb.SlackInteractionController do
+  @moduledoc """
+  This module will be responsible for consuming
+  / commands from Slack.
+  """
+  use ReleaseNotesBotWeb, :controller
+  alias ReleaseNotesBot.Projects
+  alias ReleaseNotesBotWeb.CaptainsView
+
+  @spec ping(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def ping(conn, _params) do
+    render(conn, "ping.json")
```

Comment:
What is `ping.json` ? I dont see that file anywhere?

Reply:
Good catch. Nuking this.